### ISSUE TITLE
SSL error

### ISSUE THREAD

**Issue by TimeRider07** [Nordaaker/convos#496]
Sometimes, I get this error even though the port is SSL;

"NON-SSL command received on SSL-only port. Check your connection settings."

What could it be?

**Comment by jhthorsen** [comment]
I have no idea. See https://github.com/Nordaaker/convos/issues/497#issuecomment-647214370

**Comment by TimeRider07** [comment]
Actually I am running Convos in another VPS. So my IRC server and Convos are on different VPSs. Could that be the case?

**Comment by jhthorsen** [comment]
I don't know.

**Comment by jhthorsen** [comment]
Btw: You still haven't given me any way to reproduce this error.

**Comment by TimeRider07** [comment]
If you can again go to:
https://chatsansar.com/cloud
Then, add server: chat.chatsansar.com
Port (ssl): 6697

**Comment by jhthorsen** [comment]
I don't see that message you're mentioning anywhere. When you submit a bug report you have to give this information:

1. What are you doing? (Such as connection information, commands, what did you click on, urls, ...)
2. What happens? (What do you see, where do you see it, ...)
3. What did you expect to happen? (Somehow reverse of #2).

Without sufficient information, it just ends up in guessing game and then a ticket closed as "Cannot reproduce" or "Invalid".

**Comment by TimeRider07** [comment]
okay jhthorsen. Here is in detail of what happened;
I signed up in my convos client https://chatsansar.com/convos as harish@gmail.com
Then, I went to "Connection" setting and clicked "update" as it showed I was disconnected even though I was not.
Then, it throws SSL errors;
Screenshot: https://chatsansar.com/convos/convo-ssl-error.JPG

**Comment by jhthorsen** [comment]
You have to tick the "Secure connection" checkbox if you want a secure connection.

**Comment by TimeRider07** [comment]
Ticking the ssl option made the problem go away in that particular account. But, I tried to sign up as new user in my convos and again I don't see the user list. It is weird problem going on and off.
Do I still have to tick the ssl option on the connection setting? As I had already made the default connection on my admin panel;
irc://irc.chatsansar.com:6697/%23Nepal

This would make it default ssl connection as port 6697 is generally ssl port in ircd.

And, I see a disconnected line in the bottom of the screen, is it normal?
https://chatsansar.com/convos/disconnected.JPG

Thank you!
Regards,
TimeRider
